src/components/forms/ScrollableConfigurationTable.tsx

```diff
@@ -20,7 +20,7 @@ const ScrollableConfigurationTable: FC<Props> = ({
   return (
     <ScrollableTable
       dependencies={[notify.notification]}
-      belowId="form-footer"
+      belowIds={["form-footer"]}
```

Comment:
We sometimes do need to encounter for the status bar here as well. I.e. on instance creation > network and others.